study.md

```diff
@@ -21,18 +21,20 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+A callback lets you put in a function to be called later so it doesn't run right away?
+Why would I want to do a callback? If code has to wait for input before running e.g. a 'click'?
```

Comment:
A callback function is just a function that gets passed into another function as an argument.  Here's an example.
```javascript
function greet(greeter, name) {
  console.log(`Executing greeter callback with ${name} as the argument.`);

  greeter(name);

  console.log('Greeting finished.');
}

function hello(name) {
  console.log(`Hello, ${name}.`);
}

function hi(name) {
  console.log(`Hi, ${name}.`);
}

greet(hello, 'Cathy');
// Output:
// Executing greeter callback with Cathy as the argument.
// Hello, Cathy.
// Greeting finished.

greet(hi, 'Cathy');
// Output:
// Executing greeter callback with Cathy as the argument.
// Hi, Cathy.
// Greeting finished.
```